programs/management/commands/transfer.py

```diff
@@ -0,0 +1,90 @@
+from django.core.management.base import BaseCommand
+from django.core.exceptions import ObjectDoesNotExist
+from django.db import transaction
+from programs.models import Program
+from screener.models import WhiteLabel
+from configuration.white_labels import white_label_config
+
+
+class Command(BaseCommand):
+    help = "Transfer programs from one white label to another"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "target_white_label",
+            type=str,
+            help="The white label code to transfer programs to target whilte label",
+        )
+        parser.add_argument(
+            "external_names",
+            nargs="+",
+            type=str,
+            help="External names of programs to transfer",
+        )
+
+    def validate_white_label(self, white_label_code):
+        """Validate white label exists in config and database"""
+        if white_label_code not in white_label_config:
+            self.stdout.write(
+                self.style.ERROR(
+                    f"White label '{white_label_code}' not found in configuration. "
+                    f"Please run: python manage.py add_config {white_label_code}"
+                )
+            )
+            return False
+
+        try:
+            WhiteLabel.objects.get(code=white_label_code)
+            return True
+        except ObjectDoesNotExist:
+            self.stdout.write(
+                self.style.ERROR(
+                    f"White label '{white_label_code}' not found in database. "
+                    f"Please ensure it's configured properly."
+                )
+            )
+            return False
+
+    @transaction.atomic
+    def handle(self, *args, **options):
+        target_code = options["target_white_label"]
+        external_names = options["external_names"]
+
+        # Validate white label exists
+        if not self.validate_white_label(target_code):
+            return
+
+        target_white_label = WhiteLabel.objects.get(code=target_code)
+
+        for external_name in external_names:
+            try:
+                # Get source program
+                source_program = Program.objects.get(external_name=external_name)
+
+                # Check if program already exists
+                new_external_name = f"{target_code}_{external_name.split('_')[-1]}"
```

Comment:
We don't have a consistent format for the external names, so lets just leave it empty replace it manually in the admin latter. Also, it will need an index because if there are multiple they need to be unique. Maybe add a message at the end reminding the user to add the external name.
```suggestion
                new_external_name = None
```